challenges/objects-arrays.js

```diff
@@ -58,12 +90,15 @@ The resulting contact information should have a space between the first name and
 Name email@example.com
 
 Log the result of your new array. */
-const contactInfo = [];
+const contactInfo = graduates.map(graduate => `${graduate.first_name} ${graduate.email}`)
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+
+graduates
+.forEach(element => element.university.match(/Uni/g) ? uni.push(element.university) : "");
```

Comment:
Fancy way to achieve this one! Not sure whether you wanted to push yourself, but try to research `includes()` method. Other than that - this is cool! 😄